cmd/subnetcmd/create.go

```diff
@@ -283,12 +285,12 @@ func createSubnetConfig(cmd *cobra.Command, args []string) error {
 				if err != nil {
 					return err
 				}
-				allocateToNewKeyOption := "1m to new key (+x amount to relayer)"
-				allocateToEwoqOption := "1m to new ewoq (not recommended for production, +x amount to relayer)"
-				customAllocationOption := "Custom allocation (configure exact amount to relayer)"
+				allocateToNewKeyOption := "Allocate 1m tokens to new a newly created address (Recommended for testing)"
+				allocateToEwoqOption := "Allocate 1m to the ewoq address 0xF0f...12f (Only recommended for local testing, not recommended for production)"
```

Comment:
ewoq is 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC

Reply:
ewoq can be used on local testing, devnet testing, and fuji testing with caution, surely not on mainnet